algebra/field-assembly/src/lib.rs

```diff
@@ -328,6 +328,6 @@ mod tests {
     #[test]
     fn expand_muls() {
         let impl_block = super::generate_impl(4, true);
-        println!("{}", impl_block);
+        log::debug!("{}", impl_block);
```

Comment:
If you use `log` in test you should init the backend before.... otherwise it doesn't have any sense... maybe in tests print is better.